src/user/table/index.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+import { Header } from 'user/header'
+import { Row } from 'user/row'
+import { useStyle } from 'user/style'
+import { ids } from 'user/table/id'
+
+export const Table = () => <div className={useStyle().main}>
+  <Header />
+  {ids.map((ids, index) => <Row key={index} />)}
```

Comment:
The `ids` parameter in our lambda function is not used and we use the index instead.

We can change our lambda to resemble something like this: 
```js
id => <Row key={id} />
```

Reply:
Also, we can extract this map to a different component (named something like `Content`)

Reply:
Done